scripts/pylib/twister/twisterlib.py

```diff
@@ -730,8 +730,12 @@ def handle(self):
         ser_pty_process = None
         if serial_pty:
             master, slave = pty.openpty()
+            serial_cmd = re.split(',| ', serial_pty)
+            # We try to pass the build directory to the device serial pty for specially purpose
+            serial_cmd.append(self.build_dir)
+
             try:
-                ser_pty_process = subprocess.Popen(re.split(',| ', serial_pty), stdout=master, stdin=master, stderr=master)
+                ser_pty_process = subprocess.Popen(serial_cmd, stdout=master, stdin=master, stderr=master)
             except subprocess.CalledProcessError as error:
                 logger.error("Failed to run subprocess {}, error {}".format(serial_pty, error.output))
```

Comment:
serial_pty --> serial_cmd

Reply:
Hi @chen-png,  it's serial_cmd already.  I guess you mean the removed old one. :)

Reply:
I mean you should use serial_cmd too in the looger.error.

Reply:
ok, got your point, thank for correcting it.

Reply:
@chen-png , got it fixed, thanks!